netsim/providers/__init__.py

```diff
@@ -378,11 +379,48 @@ def get_forwarded_ports(node: Box, topology: Box) -> list:
 """
 validate_images -- check the images used by individual nodes against provider image repo
 """
-
 def validate_images(topology: Box) -> None:
   p_cache: dict = {}
 
   for n_data in topology.nodes.values():
     execute_node('validate_node_image',n_data,topology)
 
   log.exit_on_error()
+
+"""
+validate_mgmt_ip -- Validate management IP addresses
+"""
+def validate_mgmt_ip(
+      node: Box,
+      provider: str,
```

Comment:
```provider``` is only used as ```module``` parameter in logging, perhaps declare this to be part of the ```initial``` module (or ```addressing``` module, although that doesn't really exist) and leave out the ```provider``` parameter?

Reply:
Looking at ```validate_pools``` in ```addressing.py``` which uses ```"addressing"```

Reply:
We need the provider parameter for the first error message. Libvirt requires IPv4 management address whereas (in theory) clab could work just with IPv6 ones.